lib/generators/boring/webmock/install/install_generator.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module Boring
+  module Webmock
+    class InstallGenerator < Rails::Generators::Base
+      desc "Adds webmock gem to the application"
+
+      SUPPORTED_TEST_FRAMEWORKS = %w[rspec minitest]
+
+      class_option :test_gem, type: :string,
+                   desc: "Tell us the framework you use for writing tests in your application. Supported options are #{SUPPORTED_TEST_FRAMEWORKS}"
+      
+      def verify_presence_of_supported_test_gem
+        test_gem = options[:test_gem]
+
+        raise(NotImplementedError, "test_gem option cant be blank! Supported test_gem options are #{SUPPORTED_TEST_FRAMEWORKS}") if test_gem.blank?
+
+        if SUPPORTED_TEST_FRAMEWORKS.exclude?(test_gem)
+          raise(NotImplementedError, "#{test_gem} is not supported as a test_gem option! Supported options are #{SUPPORTED_TEST_FRAMEWORKS}")
+        elsif test_gem.eql?('rspec')
+          verify_presence_of_test_gem('rspec-rails')
+        elsif test_gem.eql?('minitest')
```

Comment:
Let's update to just if and else since we have already handled the case of test gem invalid case above this code.

```
if test_gem.eql?('rspec')
 ....
else
  verify_presence_of_test_gem('minitest')
end
```